keps/sig-node/2008-forensic-container-checkpointing/README.md

```diff
@@ -292,14 +329,94 @@ Checkpointing containers will be possible again.
 
 ###### Are there any tests for feature enablement/disablement?
 
-Currently no.
+Currently the test will automatically be skipped if the feature is not enabled.
```

Comment:
This question is about whether or not we are testing the feature enablement/disablement. 

node-e2e tests are kinda difficult to test feature enabliement/disablement but it would be worth considering what code paths are you testing if the feature is turned off.

Its an optional endpoint in kubelet but did you have any code that the behavior changes based on the feature toggle.

Reply:
> node-e2e tests are kinda difficult to test feature enabliement/disablement but it would be worth considering what code paths are you testing if the feature is turned off.

Let's get a test case created that can detect when the feature isn't functional, but when it is functional usefully tests the feature.

Reply:
Yea, so I think you are saying that we should have a test case to see what happens if you hit the kubelet api with the feature off? Essentially making sure that if the feature is disabled we are not allowing checkpoints.

I think we could do something like in the test.

Reply:
> Let's get a test case created that can detect when the feature isn't functional, but when it is functional usefully tests the feature.

That sounds easy. Will do.

Reply:
done